backend/python/requirements.txt

```diff
@@ -68,3 +68,4 @@ uritemplate==3.0.1
 urllib3==1.26.4
 Werkzeug==1.0.1
 zipp==3.4.1
+pytz==2022.7.1
```

Comment:
This import isn't needed in the requirements.txt file because it's already imported above (in fact, importing it twice causes a build error). Also did you even end up using pytz? I noticed that you imported timezone from both pytz and datetime in the `sign_in_logs_service.py` and I'm not sure which one is being used.